### ISSUE TITLE
using tab to indent or outdent list items only indents or outdents first item in list, leaves the rest where they started

### ISSUE THREAD

**Issue by jonathanmorgan** [SublimeText-Markdown/MarkdownEditing#254]
Hello,

With Sublime Text 3 (build 3083) and the latest version of MultimarkdownEditing (2.1.1 according to package control), when I select text that contains one or more list items, then press tab to indent or shift-tab to outdent the text:

- if the first line in the selection is a list item, it and it alone is indented or outdented.  The rest of the selection stays where it is.
- if the first line in the selection is not a list item, nothing happens when you press tab or shift-tab.

This behavior only happens with markdown files.  Other syntaxes are fine (python, Java, etc.).

Any idea what is going on?  Can you reproduce?  Let me know if you want a copy of a file I'm seeing this behavior in to test on.

Thanks,

Jon

**Comment by felixhao28** [comment]
I cannot reproduce. Does this happen as well if you test it on a new file?

**Comment by jonathanmorgan** [comment]
Interesting.  It does happen with a new file, and with all files where the editing context (or whatever you call the thing in the lower right corner that tells you the type of the file - I refer to it as editing context for the remainder of the email - sorry I don't know terminology) is set to either MarkdownEditing-->Markdown, MarkdownEditing-->Markdown GFM, or MarkdownEditing-->Multimarkdown.

For example, I created a file with the following text that has the problem for me:

    - tesakhlrsa
    
    - teagesaheagea
    
        - gesgsdhsahes
    
    Grsighjsklagjdslka
    
    - gdsahsahrsfagrsa

Place these 9 lines in new file, set editing context to either MarkdownEditing-->Markdown, MarkdownEditing-->Markdown GFM, or MarkdownEditing-->Multimarkdown, select lines 3 through 9, then press tab.  On my computers (this is happening across 4 machines - 1 ubuntu 14.10, 2 OSX 10.10.2 macs, and 1 windows 8.1 machine, all running latest Sublime Text 3 and MarkdownEditing plugin), only the first line indents, the rest stay stationary.

It only happens when the editing context is set to either MarkdownEditing-->Markdown, MarkdownEditing-->Markdown GFM, or MarkdownEditing-->Multimarkdown.

I tried to attach the actual file, but I am having trouble figuring out how to attach a file other than an image for some reason.  I was able to reproduce by just copying the 9 lines above and pasting them into a new file.

On the linux machine, before upgrading to latest Sublime 3, I also tried it on Sublime 3 build 3065 with MultimarkdownEditing the only plugin installed via PackageControl, and problem was there also.
